manifests/bootkube/config/bootstrap-config-overrides.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kubecontrolplane.config.openshift.io/v1
+kind: KubeControllerManagerConfig
+serviceServingCert:
+  certFile: service-signer.crt
+extendedArguments:
+  cloud-config:
+  - "/etc/kubernetes/cloud/config"
+  root-ca-file:
+  - "/etc/kubernetes/secrets/root-ca.crt"
+  service-account-private-key-file:
+  - "/etc/kubernetes/secrets/service-account.key"
+  cluster-signing-cert-file:
+  - "/etc/kubernetes/secrets/kube-ca.crt"
+  cluster-signing-key-file:
+  - "/etc/kubernetes/secrets/kube-ca.key"
+  kubeconfig:
```

Comment:
turns out that for some reason we have to pass this on the command line in the pod for now.  don't know why.  Probably loading order.

Reply:
good to know.